locksmith/src/models/receiptsBase.ts

```diff
@@ -21,6 +22,11 @@ export class ReceiptsBase extends Model<
   declare state: string
   declare country: string
 
+  // lock & transaction details
+  declare network: number
+  declare lockAddress: string
```

Comment:
We may not need the lockAdress because we can look up by transaction but I think it;s ok!